cmd/clients.go

```diff
@@ -48,9 +56,15 @@ func PrepareClients(config config.Clients) *Clients {
 		EVMClients[configChainId].SetChainID(clientChainId.Uint64())
 	}
 
+	logger := config.GetLoggerFor("Clients")
+
```

Comment:
We do not need logging here. We use the Clients struct only to initialise stuff.